### ISSUE TITLE
Import existing temp data from InfluxDB

### ISSUE THREAD

**Issue by ironicbadger** [AnalogJ/scrutiny#33]
I have years worth of temp data for each drive stored in InfluxDB which was gathered using Telegraf. Perhaps it would be possible to sample this data at a low resolution to inform the temp graphs used here?

**Comment by AnalogJ** [comment]
yeah, it was pointed out to me that smartd stores SMART attribute data in csv files. Adding the ability to import historical SMART data for drives makes alot of sense. I think that mechanism should be flexible enough to also import temperature data. 

https://github.com/AnalogJHQ/scrutiny/issues/12

**Comment by ironicbadger** [comment]
If you consider this as a dupe, I won't be upset if you close.

**Comment by AnalogJ** [comment]
I'll keep it as a linked issue. Once I start prioritizing post-v1.0 issues, I'll probably consolidate these and other similar requests into one import mechanism.

**Comment by kaysond** [comment]
This issue is being closed automatically as a part of an effort to streamline and prioritize development effort. If it is still relevant, please reply to this comment and we will convert it to a discussion.
